source/css/_common/components/highlight/copy-code.styl

```diff
@@ -1,47 +1,69 @@
+.highlight-wrap {
+  position: relative;
+}
+
+.highlight-wrap:hover .copy-btn,
+.highlight-wrap .copy-btn:focus {
+  opacity: 1;
+}
+
 .copy-btn {
   display: inline-block;
-  padding: 6px 12px;
+  position: absolute;
+  padding: 2px 6px;
   font-size: 13px;
   font-weight: 700;
   line-height: 20px;
   color: #333;
   white-space: nowrap;
   vertical-align: middle;
   cursor: pointer;
-  if hexo-config('codeblock.copy_button.style') == 'flat' {
```

Comment:
`===` is better?

Reply:
Nope, there is no `===` operator in this language 😂

Reply:
Really? 🤨

Reply:
Yes, you can try 🤔